<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bài tập</string>
  </property>
  <widget class="QLabel" name="lbl_socau">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>372</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Số câu hỏi:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tao">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tạo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thoat">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_socau">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_tl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>427</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tự luận</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_tn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>407</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trắc nghiệm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_loai">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>406</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loại câu hỏi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ll_dang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chọn dạng:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tbr_mota">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mota">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mô tả:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listw_dang">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_thoat</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
